README.md

```diff
@@ -99,6 +99,7 @@ For the moment, we are currently focused on the CISA KEV Database and Google Tsu
 
 | CVE ID                                                  | Implemented | Details                                                 |
 |---------------------------------------------------------|-------------|---------------------------------------------------------|
+| CVE-2023-27524                                          | ✅           | Official Nuclei template.                               |
```

Comment:
can you please put it in the correct order

Reply:
The order in the list is not consistent

Reply:
only the first 9 elements are not ordered , I think they are the ones that you added at the end , they were : ⌛

Reply:
CVE-2023-40044 was published in 09/27/2023
CVE-2023-41061 was published in 09/07/2023

In the list CVE-2023-41061 is considered more recent than CVE-2023-40044.

I'm fixing the order in a separate PR.